rpc/rpc_machine.c

```diff
@@ -796,7 +796,7 @@ static void net_buf_received(struct m0_net_buffer    *nb,
 	m0_rpc_packet_init(&p, machine);
 	rc = m0_rpc_packet_decode(&p, &nb->nb_buffer, offset, length);
 	if (rc != 0)
-		M0_LOG(M0_ERROR, "Packet decode error: %i.", rc);
+		M0_LOG(M0_DEBUG, "Packet decode error: %i.", rc);
```

Comment:
Why?

Reply:
During ST testing, we observed that the logs were filled up with these error logs during the rdma read operation where we send a dummy msg with dummy data to notify the remote target about the completion.
Reverted to M0_ERROR level.